guava/src/com/google/common/collect/Iterables.java

```diff
@@ -786,8 +783,7 @@ public void forEach(Consumer<? super T> action) {
    * @return the first element of {@code iterable} or the default value
    * @since 7.0
    */
-  @NullableDecl
-  public static <T> T getFirst(Iterable<? extends T> iterable, @NullableDecl T defaultValue) {
+  public static <T extends @Nullable Object> @PolyNull T getFirst(Iterable<? extends T> iterable, @PolyNull T defaultValue) {
```

Comment:
I don't understand this comment.

What does "as per the stub file" mean?  If it's in the stub file, and this pull request is for the stub file, then why isn't it already in the diffs?

Why do you think `@Nullable` will be more appropriate?  Please explain your reasoning, or I cannot comment on it.

Reply:
By "as per the stub file", I meant the annotations that are specified in the stub file for the given class. 

I have added the annotations as you can see in the diff.

I was just thinking if this could be replaced with @Nullable annotation, for this method is just returning value from another method which is: `public static <T> @Nullable T getNext(Iterator<? extends T> iterator, @Nullable T defaultValue)`. And return type of that method is annotated to be @Nullable, so I thought it might be better annotated as @Nullable. But I haven't made this change, I just wanted to have your opinion on this. Thanks.